code/modules/client/preference_setup/loadout/lists/utility.dm

```diff
@@ -36,6 +36,10 @@
 	display_name = "cheap tablet computer"
 	path = /obj/item/modular_computer/tablet/preset/custom_loadout/cheap
 	cost = 2
+/datum/gear/utility/wheelchair
+	display_name = "wheelchair"
+	path = /obj/structure/bed/chair/wheelchair
```

Comment:
Replace with the foldable wheelchair object

Reply:
The path must be a item